cloudify-services/values.yaml

```diff
@@ -103,16 +68,15 @@ composer_frontend:
       successThreshold: 1
       failureThreshold: 3
 
-
 execution_scheduler:
   replicas: 1
   image: 263721492972.dkr.ecr.eu-west-1.amazonaws.com/cloudify-manager-execution-scheduler:latest-x86_64
   imagePullPolicy: IfNotPresent
 
 fileserver:
   install: yes
-  access_key: admin  # e.g. username
-  secret_key: admin123  # e.g. password
+  access_key: admin
```

Comment:
I'd leave the comments (as `access_key`/`secret_key`) are not obvious places to put username/password to.